src/shared/components/case-editor/case-edit-submit/case-edit-submit.html

```diff
@@ -91,7 +91,7 @@ <h2 class="heading-h2">Check your answers</h2>
   </ng-container>
   <div class="form-group form-group-related">
     <button class="button button-secondary" type="button" [disabled]="!hasPrevious()" (click)="previous()">Previous</button>
-    <button type="submit" [disabled]="isDisabled()" class="button">{{triggerText}}</button>
+    <button type="submit" [disabled]="isDisabled() || isSubmitting" class="button">{{triggerText}}</button>
```

Comment:
we could perhaps introduce this check inside that isDisabled() method itself?

Reply:
actually good point, refactored